File: QTD2.Domain/Entities/Core/Version_EnablingObjective_Procedure_Link.cs
namespace QTD2.Domain.Entities.Core
{
    public class Version_EnablingObjective_Procedure_Link : Common.Entity
    {
        public int Version_EnablingObjectiveId { get; set; }

        public int Version_ProcedureId { get; set; }

        public virtual Version_EnablingObjective Version_EnablingObjective { get; set; }

        public virtual Version_Procedure Version_Procedure { get; set; }

        public Version_EnablingObjective_Procedure_Link()
        {
        }

        public Version_EnablingObjective_Procedure_Link(int version_EnablingObjectiveId, int version_ProcedureId)
        {
            Version_EnablingObjectiveId = version_EnablingObjectiveId;
            Version_ProcedureId = version_ProcedureId;
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/Version_EnablingObjective_Procedure_Link/Version_EnablingObjective_Procedure_Link.ts
import { Entity } from '../Entity';
import { Version_EnablingObjective } from '../Version_EnablingObjective/Version_EnablingObjective';
import { Version_Procedure } from '../Version_Procedure/Version_Procedure';

export class Version_EnablingObjective_Procedure_Link extends Entity {
  version_EnablingObjectiveId!: any;
  version_ProcedureId!: any;
  version_EnablingObjective!: Version_EnablingObjective;
  version_Procedure!: Version_Procedure;
}
